### ISSUE TITLE
transitive dependency support

### ISSUE THREAD

**Issue by trevjonez** [Sunzxyong/external-plugin-support#1]
com.android.tools.build:gradle:3.2.1 has many transitive deps that would be nice to see loaded in the list as well.

[gradle-3.2.1.pom](https://github.com/Sunzxyong/external-plugin-support/files/2480893/gradle-3.2.1.copy.txt)

**Comment by Sunzxyong** [comment]
@trevjonez Yes, but many plugins have many java library dependencies, and for gradle plugins, we focus more on the functionality of the plugins themselves.
